lib/onboarding/gen_phrase/cubit/onboarding_gen_phrase_cubit.dart

```diff
@@ -35,7 +35,7 @@ class OnBoardingGenPhraseCubit extends Cubit<OnBoardingGenPhraseState> {
     try {
       emit(state.copyWith(status: OnBoardingGenPhraseStatus.loading));
       final mnemonicFormatted = mnemonic.join(' ');
-      await saveMnemonicKey(mnemonicFormatted);
+      await secureStorageProvider.set('mnemonic', mnemonicFormatted);
```

Comment:
@bibash28 
Please use "SecureStorageKey.mnemonic" instead of the hardcoding string key